### ISSUE TITLE
Subpleural scarring

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6571]
**Preferred term label:** Subpleural scarring 
**Synonyms**
Subpleural interstitial thickening
Subpleural fibrosis
**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Pulmonary fibrosis HP:0002206
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Can we provide a more detailed description?
For instance, how to identify this finding in https://err.ersjournals.com/content/22/129/292.long (Fig 2).

**Comment by eswietlik** [comment]
@pnrobinson Actually while browsing the ontology I found that the term already exists Subpleural interstitial thickening HP:0025178 so we can close this one, sorry for the inconvenience.

**Comment by pnrobinson** [comment]
@eswietlik  -- no worries. I added Subpleural scarring as a synonym to Subpleural interstitial thickening
